src/NServiceBus.Core.Tests/Msmq/QueuePermissionsTests.cs

```diff
@@ -66,18 +84,22 @@ public void Should_log_if_queue_exists()
             Assert.That(logOutput.ToString(), Does.Contain("Verified that the queue"));
         }
 
-        [Test]
-        public void Should_warn_if_queue_has_public_access()
+        [TestCase(MessageQueueAccessRights.FullControl, WellKnownSidType.WorldSid)]
+        [TestCase(MessageQueueAccessRights.GenericRead, WellKnownSidType.WorldSid)]
+        [TestCase(MessageQueueAccessRights.GenericWrite, WellKnownSidType.WorldSid)]
+        [TestCase(MessageQueueAccessRights.FullControl, WellKnownSidType.AnonymousSid)]
+        [TestCase(MessageQueueAccessRights.GenericRead, WellKnownSidType.AnonymousSid)]
+        [TestCase(MessageQueueAccessRights.GenericWrite, WellKnownSidType.AnonymousSid)]
+        public void Should_warn_if_queue_has_public_access(MessageQueueAccessRights rights, WellKnownSidType sidType)
```

Comment:
Same here there are more options. should we check those? (Like `DeleteQueue` https://msdn.microsoft.com/en-us/library/system.messaging.messagequeueaccessrights(v=vs.110).aspx )